sklearn/multioutput.py

```diff
@@ -660,7 +660,9 @@ def _log_message(self, *, estimator_idx, n_estimators, processing_msg):
     def _get_predictions(self, X, *, output_method):
         """Get predictions for each model in the chain."""
         check_is_fitted(self)
-        X = validate_data(self, X, accept_sparse=True, reset=False)
+        X = validate_data(
+            self, X, accept_sparse=True, reset=False, ensure_all_finite="allow-nan"
```

Comment:
Actually, I think that we can set the flag to `False` to completely delagate to the underlying estimator. If the underlying estimator is handling `inf` values, it should be fine.

I see this pattern in `BaggingRegressor` for instance.

Reply:
```suggestion
            self, X, accept_sparse=True, reset=False, ensure_all_finite=False